components/Label/Label.test.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Label } from './Label'; 
+
+const variants:Array<
+| 'weeklyPickButtonLabel'
+| 'inputLabel'
+| null
+| undefined
+> = ['weeklyPickButtonLabel', 'inputLabel'];
+
+const variantClasses = {
+  weeklyPickButtonLabel: 'text-zinc-50 cursor-pointer rounded-xl items-center py-4 px-3 border-2 border-zinc-800 peer-aria-checked:border-orange-600 peer-hover:bg-zinc-800 flex',
+  inputLabel: 'text-zinc-900 cursor-text flex-col',
+};
+
+describe('Label', () => {
+  variants.forEach((variant) => {
+    it(`renders correctly with variant ${variant}`, () => {
+      render(
+        <Label data-testid="testLabel" variant={variant}>
+          Test Label
+        </Label>
+      );
+
+      const label = screen.getByTestId('testLabel');
+      expect(label).toBeInTheDocument();
+      if (variant) {
+        expect(label).toHaveClass(variantClasses[variant]);
+    }
+    });
+  });
+
+  it(`applies additional custom classes correctly`, () => {
+    render(
+      <Label
+        data-testid="customClassLabel"
```

Comment:
This should follow our custom classes and other `data-testid` and be hyphenated, not camelCased.

Reply:
@ryandotfurrer That's one question I meant to ask: looking at the tests inside the `app` we use hyphenated values for the `data-testid` (see `login/page.tsx` and `register/Register.tsx`), but inside the `components` some are camel case (see `Button/Button.test.tsx` and `LeagueCard/LeagueCard.test.tsx`).

Hyphenated it is then!

Reply:
This is a standard that @shashilo mentioned early on to one of my PRs. He wants them hyphenated no matter where they are.